lib/JSONSchemaGenerator.js

```diff
@@ -1097,15 +1097,15 @@ function propertiesPatchCommandsValidation(
     [PULLCMD]: {
       type: 'object',
       properties: arrayPropertiesWithMongoConditions,
-      ...Object.keys(arrayPatternProperties).length > 0
+      ...Object.keys(arrayPatternPropertiesWithMongoConditions).length > 0
```

Comment:
Is there a reason for not adopting `arrayPatternPropertiesWithMongoConditions` also for the `PUSHCMD`?